docs/sphinx/source/whatsnew/v0.9.4.rst

```diff
@@ -56,4 +58,7 @@ Contributors
 * Kevin Anderson (:ghuser:`kanderso-nrel`)
 * Karel De Brabandere (:ghuser:`kdebrab`)
 * Naman Priyadarshi (:ghuser:`Naman-Priyadarshi`)
+* Anton Driesse (:ghuser:`adriesse`)
+* Cliff Hansen (:ghuser:`cwhanse`)
+* Kevin Anderson (:ghuser:`kanderso-nrel`)
```

Comment:
Cliff and I are already listed a bit farther up :)

Reply:
I guess that must have happened in the merge.